swagger.yml

```
@@ -719,6 +719,25 @@ paths:
             $ref: '#/definitions/file'
         default:
           $ref: '#/responses/error'
+  /purge:
+    post:
+      operationId: purgeSiteCache
+      tags: [purge]
+      consumes:
+        - application/json
+      parameters:
+        - name: payload
+          in: body
+          schema:
+            $ref: '#/definitions/purge'
```

idea, what if we had this so we can specify the required paramters better?
```suggestion
            oneOf:
              - $ref: '#/components/schemas/PurgeSiteID'
              - $ref: '#/components/schemas/PurgeSiteSlug'
              - $ref: '#/components/schemas/PurgeDomain'
```

---

talked about it on slack, we're running open-api 2.0 still, which doesn't support this